src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1993,6 +1994,17 @@ def load_arguments(self, _):
         c.argument("dns_zone_resource_ids", options_list=["--ids"], required=True)
         c.argument("attach_zones")
 
+    with self.argument_context('aks check-network outbound') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'],
+                   required=True, help='Name of the managed cluster.')
+        c.argument('resource_group', options_list=['--resource-group', '-g'],
+                   required=True, help='Name of the resource group.')
```

Comment:
could omit this, this has been declared at around line 340

Reply:
Is there a way to add a shorter name for `--resource-group`? I couldn't find its defintion around line 340. When I omitted the line, running the command with argument `-h` returned no description for the argument `--resource-group` and I can't no longer use the shorter name 
![image](https://github.com/Azure/azure-cli-extensions/assets/49544763/e9119593-a6ad-4d11-bdb8-2e7af0c194d0)

Reply:
see comment below

Reply:
got it!